src/Uno.PackageDiff/IDiffWriter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using NuGet.Versioning;
+using Mono.Cecil;
+
+namespace Uno.PackageDiff
+{
+	internal interface IDiffWriter : IDisposable
```

Comment:
this contract looks highly tied to xml syntax (`WriteXyzStart`, `WriteXyzEnd`), and md formatting (XyzStart that takes in an `int`).
it should instead accept the array of missing types, and (containing types + their missing members: properties,fields,method and events)
or anything more context-relevant